packages/protocol-kit/src/contracts/AbiType/SafeProxyFactory/safeProxyFactoryContract.ts

```diff
@@ -0,0 +1,98 @@
+import {
```

Comment:
File name first letter should be uppercased right?

safeProxyFactoryContract.ts -> SafeProxyFactoryContract.ts